### PR TITLE
docs/packaging: Update release process for package (un)deprecation

### PR DESCRIPTION
**Pull request by joebonrichie**
Depends on
https://github.com/getsolus/packages/pull/2900
https://github.com/getsolus/solus-sc/pull/341

### CODE REVIEW

**Review by ermo — APPROVED** [review]
LGTM!

File: docs/packaging/procedures/release-processes.md [hunk 1]
```diff
@@ -33,23 +33,24 @@ Deprecated packages will remove themselves from the users systems as the first o
 
 Follow these steps to submit a package for deprecation:
 
-1. Fork and clone the `solus-sc` repo [on GitHub](https://github.com/getsolus/solus-sc), and/or create a new branch
+1. Create a new branch on the monorepo [on GitHub](https://github.com/getsolus/packages) with a suitable name e.g. deprecate-foobar
```
Comment (Staudey) [inline docs/packaging/procedures/release-processes.md:36]:
```suggestion
1. Create a new branch in the monorepo [on GitHub](https://github.com/getsolus/packages) with a suitable name, for example deprecate-foobar
```
I think the guideline is to use no Latin abbreviations like "e.g." and "i.e."

Should there be a note to fork it, for external contributors?

File: docs/packaging/procedures/release-processes.md [hunk 2]
```diff
@@ -33,23 +33,24 @@ Deprecated packages will remove themselves from the users systems as the first o
...
 2. Edit `repo_data/distribution.xml.in`
 3. Add the package you wish to deprecate at the end of the `<Obsoletes>` section
-4. Run the `merged_repos.sh` script
+4. Change directory to repo_data (`cd repo_data`), then run the `merged_repos.sh` script
+5. Remove the package from the monorepo e.g. `rm -fr packages/f/foobar`
```
Comment (Staudey) [inline docs/packaging/procedures/release-processes.md:40]:
```suggestion
5. Remove the package from the monorepo, for example `rm -fr packages/f/foobar`
```

File: docs/packaging/procedures/release-processes.md [hunk 3]
```diff
@@ -33,23 +33,24 @@ Deprecated packages will remove themselves from the users systems as the first o
...
-
 ### Un-deprecating a package
 
 The above steps also apply to un-deprecating a package, with the following additions:
 
-1. Before making changes, the package repository must be reactivated
-2. In Step 2, remove the package from the list instead of adding it
-3. After the `solus-sc` pull request has been merged, ask a Solus Staff member to use available tooling to update the repository files, then index the repo with `ferryctl index unstable` (or do it yourself if you have the ability)
-4. Bump or update the package
+1. Create a new branch on the monorepo [on GitHub](https://github.com/getsolus/packages) with a suitable name e.g. undeprecate-foobar
```
Comment (Staudey) [inline docs/packaging/procedures/release-processes.md:48]:
```suggestion
1. Create a new branch in the monorepo [on GitHub](https://github.com/getsolus/packages) with a suitable name, for example undeprecate-foobar
```

File: docs/packaging/procedures/release-processes.md [hunk 4]
```diff
@@ -33,23 +33,24 @@ Deprecated packages will remove themselves from the users systems as the first o
...
+2. Find the commit deprecating the package and restore the package within the monorepo structure. e.g. Restore all files in packages/f/foobar or use `git revert` if appropriate
```
Comment (Staudey) [inline docs/packaging/procedures/release-processes.md:49]:
```suggestion
2. Find the commit deprecating the package and restore the package within the monorepo structure. For example restore all files in packages/f/foobar or use `git revert` if appropriate
```

File: docs/packaging/procedures/release-processes.md [hunk 5]
```diff
@@ -33,23 +33,24 @@ Deprecated packages will remove themselves from the users systems as the first o
...
+3. Edit `repo_data/distribution.xml.in` and remove the package you wish to undeprecate in the `<Obsoletes>` section
+4. Change directory to repo_data (`cd repo_data`), then run the `merged_repos.sh` script
+5. Open a Pull Request on GitHub and ask a Solus Staff member to use available tooling to update the repository files, then index the repo with `ferryctl index unstable` (or do it yourself if you have the ability)
```
Comment (Staudey) [inline docs/packaging/procedures/release-processes.md:52]:
```suggestion
5. Open a Pull Request on GitHub and ask a Solus Staff member to use available tooling to update the repository files, then index the repository with `ferryctl index unstable` (or do it yourself if you have the ability)
```

File: docs/packaging/procedures/release-processes.md [hunk 6]
```diff
@@ -33,23 +33,24 @@ Deprecated packages will remove themselves from the users systems as the first o
 
 Follow these steps to submit a package for deprecation:
 
-1. Fork and clone the `solus-sc` repo [on GitHub](https://github.com/getsolus/solus-sc), and/or create a new branch
+1. Create a new branch in the monorepo [on GitHub](https://github.com/getsolus/packages) (forking the repository first if necessary) with a suitable name such as deprecate-foobar
```
Comment (Staudey) [inline docs/packaging/procedures/release-processes.md:36]:
```suggestion
1. Create a new branch in the Solus `packages` repository [on GitHub](https://github.com/getsolus/packages) (forking it first if necessary) with a suitable name such as "deprecate-foobar"
```

I remembered that "monorepo" is also a no-no

File: docs/packaging/procedures/release-processes.md [hunk 7]
```diff
@@ -33,23 +33,24 @@ Deprecated packages will remove themselves from the users systems as the first o
...
 2. Edit `repo_data/distribution.xml.in`
 3. Add the package you wish to deprecate at the end of the `<Obsoletes>` section
-4. Run the `merged_repos.sh` script
+4. Change directory to repo_data (`cd repo_data`), then run the `merged_repos.sh` script
+5. Remove the package from the monorepo, for example `rm -fr packages/f/foobar`
```
Comment (Staudey) [inline docs/packaging/procedures/release-processes.md:40]:
```suggestion
5. Remove the package from the Solus `packages` repository, for example `rm -fr packages/f/foobar`
```

File: docs/packaging/procedures/release-processes.md [hunk 8]
```diff
@@ -33,23 +33,24 @@ Deprecated packages will remove themselves from the users systems as the first o
...
-
 ### Un-deprecating a package
 
 The above steps also apply to un-deprecating a package, with the following additions:
 
-1. Before making changes, the package repository must be reactivated
-2. In Step 2, remove the package from the list instead of adding it
-3. After the `solus-sc` pull request has been merged, ask a Solus Staff member to use available tooling to update the repository files, then index the repo with `ferryctl index unstable` (or do it yourself if you have the ability)
-4. Bump or update the package
+1. Create a new branch in the monorepo [on GitHub](https://github.com/getsolus/packages) (forking the repository first if necessary) with a suitable name such as undeprecate-foobar
```
Comment (Staudey) [inline docs/packaging/procedures/release-processes.md:48]:
```suggestion
1. Create a new branch in the Solus `packages` repository [on GitHub](https://github.com/getsolus/packages) (forking it first if necessary) with a suitable name such as "undeprecate-foobar"
```

File: docs/packaging/procedures/release-processes.md [hunk 9]
```diff
@@ -33,23 +33,24 @@ Deprecated packages will remove themselves from the users systems as the first o
...
+2. Find the commit deprecating the package and restore the package within the monorepo structure. For example restore all files in packages/f/foobar or use `git revert` if appropriate
```
Comment (Staudey) [inline docs/packaging/procedures/release-processes.md:49]:
```suggestion
2. Find the commit deprecating the package and restore the package within the repository structure. For example restore all files in packages/f/foobar or use `git revert` if appropriate
```

**Pull request merged**